numba/tests/test_unicode.py

```diff
@@ -337,6 +349,41 @@ def test_find(self, flags=no_pyobj_flags):
                                  cfunc(a, substr),
                                  "'%s'.find('%s')?" % (a, substr))
 
+    def test_rfind(self):
+        pyfunc = rfind_usecase
+        cfunc = njit(pyfunc)
+
+        for s in UNICODE_EXAMPLES:
+            extras = ['', 'xx', s[:-2], s[3:], s]
+            for sub_str in extras:
+                self.assertEqual(pyfunc(s, sub_str),
+                                 cfunc(s, sub_str),
+                                 "'%s'.rfind('%s')?" % (s, sub_str))
+
+    def test_rfind_with_start_end(self):
```

Comment:
These two tests have the same issues as those in https://github.com/numba/numba/pull/4204

 * https://github.com/numba/numba/pull/4204#discussion_r306190873
* https://github.com/numba/numba/pull/4204#discussion_r306191707

Reply:
all the suggestions for count method applies to rfind method as well. So I will make the necessary changes here.